app/serializers/api/admin/overridable_product_serializer.rb

```diff
@@ -0,0 +1,16 @@
+class Api::Admin::OverridableProductSerializer < ActiveModel::Serializer
+  attributes :id, :name
```

Comment:
are you sure none of the other attributes are needed in the inventory page? like :sku or :variant_unit?
just asking.

Reply:
I've just double-checked and we don't need these attributes on products here, as far as I can tell. This page is all about variants.

Reply:
I've added an extra commit to remove another two unused methods from the `ProductSimpleSerializer`, it's now slimmer.